docs/quickstart.rst

```diff
@@ -87,7 +116,12 @@ Once you have a recent Python 3 version of ``pip``, you should be able to upgrad
 
 .. code-block:: bash
 
-    pip install --upgrade setuptools
+    $ pip install --upgrade setuptools
+
+    OR
+
+    $ sudo pip3 install --upgrade setuptools
```

Comment:
Why does this one use sudo but the one above doesn't? Same comment for the next subsection.